apollo-router-core/src/services/mod.rs

```diff
@@ -168,27 +171,28 @@ impl RouterRequest {
             .uri(uri)
             .method(method)
             .body(gql_request)
-            .build()
-            .expect("body is always valid qed");
+            .build()?;
```

Comment:
Ah, that answers my earlier question. It feels like we never do anything but assume that the content is valid. If that's the case then we should `expect` here, but if there is a reason to be exposing it, then we should.

Reply:
Serialization can fail, so it would seem to be unsafe to expect here.